test.py

```diff
@@ -19,40 +22,54 @@ def download_to(url, target):
     os.rename(download_path, target)
 
 
+def validate_json(dir_path, name, fails, hubmap_schema):
+    print(f'\tLoad JSON: {name}')
+    path = os.path.join(dir_path, name)
```

Comment:
The `pathlib` module provides a much nicer interface for manipulating filesystem paths, and is worth using when possible.

Reply:
Since this is merged, I filed new issue. #22